pkg/dockercreds/export_test.go

```diff
@@ -0,0 +1,3 @@
+package dockercreds
+
+var NewCredHelperFromEnvVars = newCredHelperFromEnvVars
```

Comment:
If I'm reading this correctly, the whole point of this file is to expose the `newCredHelperFromEnvVars` function for tests? In that case a better idea is to just use `package dockercreds` in cred_helper_test.go.

The way go works, this would expose the function to _any_ tests, including ones from outside this module.

Reply:
FYI, I've found testing to be somewhat inconsistent across even files of the same project. 

The rule of thumb is that if you purely want to test public functions, then you can use `package my_module_test`, if you want to test internals, then it's perfectly fine to use `package my_module` inside a `*_test.go` file